tests/test_tts.py

```diff
@@ -85,5 +85,13 @@ def test_speak_uk(self):
         file, _ = self.tts.get_tts("Привіт.", out_file, speaker = speaker)
         self.assertEqual(file, out_file)
 
+    def test_ipython_format(self):
+        speaker = {
+            "language" : "uk"
+        }
+        out_file = os.path.join(os.path.dirname(__file__), "test.wav")
+        ipython_dict = self.tts.get_audio("Привіт.", speaker = speaker, audio_format="ipython")
+        self.assertIsInstance(ipython_dict, dict)
```

Comment:
Consider adding a type check for `ipython_dict['data']` and/or check `ipython_dict.keys()` since those should be static. This also helps prevent breaking changes in the future

Reply:
Done with https://github.com/NeonGeckoCom/neon-tts-plugin-coqui-ai/pull/6/commits/13f0531a44644dc26e90997c9e9ba3a92075f4c0